src/utils/index.ts

```diff
@@ -421,5 +421,4 @@ export const isChristmasTime = () => {
   return currentTime.month() === 11 && currentTime.date() >= 15
 }
 
-export const isSupportLimitOrder = (chainId: ChainId) =>
-  ENV_LEVEL < ENV_TYPE.PROD && NETWORKS_INFO_CONFIG[chainId].limitOrder
+export const isSupportLimitOrder = (chainId: ChainId) => NETWORKS_INFO_CONFIG[chainId].limitOrder
```

Comment:
Refactor to remove this? I think this func is unecessary

Reply:
will remove when release all chain
<img width="480" alt="image" src="https://user-images.githubusercontent.com/33005392/211970071-246a176d-778a-4b0c-b1fa-6e65f4e90885.png">

Reply:
I means, replace all those func to check `NETWORKS_INFO_CONFIG[chainId].limitOrder` directly instead of calling this func. This func is still ok, just a suggestion